packages/api/package.js

```diff
@@ -8,7 +8,6 @@ Package.describe({
 
 Npm.depends({
   accounting: '0.4.1',
-  'lodash.omit': '4.5.0',
```

Comment:
You have a good chance to keep your package free from any dependencies except for accountsjs, see my comments about lodash in the review